test/deploy.sh

```diff
@@ -208,6 +208,10 @@ if [ "$DOTEST" == "true" ]; then
    sleep 20
    check_result
    [ -e $RESULT_FILE ] && cat $RESULT_FILE
+   if [ "$err" = true ] ; then
```

Comment:
can you use 
```"$err" == "true"```
 to be consistent with previous check.

Reply:
Addresed in latest